updates (#6)
olgarithms/sphinx-tutorial: docs/9-versioning.md

### Generating different versions for your docs

Let's test this out.

> Let's tag the current version of our docs with `v0.0.1`. We'll create a tag and push it to our
> remote repository:

Reviewer: ```suggestion
We'll create a tag, `v0.0.1`, for our current version and push it to our remote repository:
```
